ui/src/map/map-view.tsx

```diff
@@ -117,6 +117,10 @@ export type MapViewProps = {
     onMapLayerChange: (change: MapLayerMenuChange) => void;
     mapLayerMenuGroups: MapLayerMenuGroups;
     visibleLayerNames: MapLayerName[];
+    publicationCandidates?: PublicationCandidate[];
+    customActiveMapTool?: MapTool;
+    designPublicationMode?: DesignPublicationMode;
+    mapTools?: SelectableMapTool[];
```

Comment:
Karttatyökalut voidaan antaa nyt ulkoapäin. Täten esim. aluevalintatyökalu on pelkästään esikatselunäkymällä